Code/GraphMol/Fingerprints/MACCS.cpp

```diff
@@ -21,7 +21,8 @@
 #include <RDGeneral/BoostEndInclude.h>
 
 namespace {
-struct Patterns {
+class Patterns {
```

Comment:
seems like it would be more readable to have the members be `std::unique_ptr<RDKit::ROMol>`. Then this could stay a struct and you wouldn't need the dtor.

Reply:
Ok, done.